packages/clarity-js/src/layout/animation.ts

```diff
@@ -27,6 +28,14 @@ export function start(): void {
         overrideAnimationHelper(animationPause, "pause");
         overrideAnimationHelper(animationCancel, "cancel");
         overrideAnimationHelper(animationFinish, "finish");
+        if (elementAnimate === null) {
+            elementAnimate = Element.prototype.animate;
+            Element.prototype.animate = function(): Animation {
+                var createdAnimation = elementAnimate.apply(this, arguments);
```

Comment:
usually we do our logic before applying (see line 71) - is it intentional to do it the other way?